### ISSUE TITLE
Configurator Stuck Loading on Kira

### ISSUE THREAD

**Issue by GnarleyCosmonaut** [kiibohd/configurator#120]
On windows 10 I am unable to access the kira page in the configuration tool. It is stuck "loading" even after refreshing, reopening, restarting the computer and trying again. I gave the tool over an hour to load and it still did not come up.

**Comment by GnarleyCosmonaut** [comment]
Getting a connection_timed_out error when trying to pull the Kira-Standard.jason file. Does not seem to be a firewall issue.

**Comment by tomplex** [comment]
I'm getting the same issue on MacOS 10.13.6.

**Comment by simplexion** [comment]
I'm seeing this issue in Windows 10 1709 as well. This is while running Kiibohd as Administrator too. IN Linux (running as root) it works fine.

**Comment by bjohnso5** [comment]
I am also having this issue, but in my case I think it IS a corporate firewall issue. Is it possible to host the layout files on standard http port 80 or even better, over https on 443?

**Comment by travVT** [comment]
I'm having the exact same issue with my Kira keyboard on Windows 10. @bjohnso5, did you ever have any luck solving this? Thanks!

**Comment by bjohnso5** [comment]
@travVT no, sorry, I didn't want to spend too much work time looking into it, as that's where I use my Kira.

**Comment by Kyle-Falconer** [comment]
Similar issue on macOS Mojave, with the K-Type keyboard.

I'm getting a `net::ERR_CONNECTION_TIMED_OUT` for "versions" and "KType-Standard.json"


![Screen Shot 2019-07-25 at 10 52 55 AM](https://user-images.githubusercontent.com/1034255/61896716-61443e80-aeca-11e9-98ef-f51700ea4d35.png)

**Comment by Kyle-Falconer** [comment]
On further investigation, I have confirmed that I'm unable to access that URL that is requested by the software:

```
http://vash.input.club:3001/versions
```


On my work laptop, I can access `vash.input.club` but not over port 3000.

On my home computer, I can get a response from that URL over 3000 just fine.

**Comment by tomplex** [comment]
For me it was my corporate network blocking certain ports. I ended up connecting my computer to my phone's wifi hotspot and re-trying; it worked just fine. You should either try to configure your keyboard at home or using some other network, if you have access.

**Comment by Kyle-Falconer** [comment]
The fix for this should be to re-deploy the API for that endpoint to a more standard port 80 or port 443. That way, it would have a much lower chance of getting blocked by corporate policies.

**Comment by Kyle-Falconer** [comment]
@jbondeson do you know how this endpoint might be able to be moved to a standard port? It seems you were the last person to set this:

https://github.com/kiibohd/configurator/blob/1ba70df9046443ac465e1c2e801025ad1f68b4cc/src/renderer/state/settings.js#L13

**Comment by Kyle-Falconer** [comment]
It's working for me now with 1.0.2 and commit 18ac1cd950fb16c0985da50340d2560a0b525022

Thanks @jbondeson !!! This might have been a "small issue" for you, but it was a total blocker for those of us affected.

**Comment by travVT** [comment]
1.0.2 is working for me too! Thanks @jbondeson. :)
